package.json

```diff
@@ -23,7 +23,13 @@
     "demo:build": "nwb build-react-component src/index.js",
     "lint": "eslint src stories",
     "prepush": "yarn lint",
-    "release": "rm -f .babelrc && yarn build && node scripts/publish-release.js"
+    "release": "rm -f .babelrc && yarn build && node scripts/publish-release.js",
+    "check": "gts check",
+    "compile": "tsc -p .",
+    "fix": "gts fix",
+    "prepare": "npm run compile",
+    "pretest": "npm run compile",
+    "posttest": "npm run check"
```

Comment:
`npm run check` throw me bunch of errors.

Reply:
Yeah I don't know what this is all about. It's stuff like obscure dependencies not being installed, but shouldn't yarn take care of that?
```
error "jsdom#request#har-validator" not installed
error "@cognite/react-scripts#uglifyjs-webpack-plugin#find-up#locate-path" not installed
error "jsdom#request#mime-types" not installed
error "jsdom#request#oauth-sign" not installed
error "h2x-parse#jsdom#request" not installed
error "jsdom#request-promise-native#tough-cookie" not installed
error "jsdom#request#tough-cookie" not installed
error "jsdom#request-promise-native#request-promise-core" not installed
error "h2x-parse#jsdom#request-promise-native" not installed
error "jsdom#har-validator#ajv" not installed
error "h2x-parse#request#aws4" not installed
error "h2x-parse#request#combined-stream" not installed
```

(random snippet)